.travis.yml

```diff
@@ -185,15 +185,22 @@ matrix:
           arch: s390x
           language: c
           dist: bionic
-          stage: Cron tests
+          stage: Final tests
```

Comment:
For debugging purpose, why not move them to initial stage so they run first?

Reply:
Duh, yes, of course!